.github/workflows/release_java.yml

```diff
@@ -0,0 +1,137 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Bindings Java Release
+
+on:
+  push:
+    tags:
+      # Staging JARs on Apache Nexus repository for RCs. Read more on
+      # https://opendal.apache.org/docs/contributing/release#release-maven-artifacts
+      - 'v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
```

Comment:
We only stage while pushing rc tags?

Reply:
See https://github.com/apache/incubator-opendal/pull/2939#discussion_r1306593129.

As Maven release cannot be revoked, and follow the release process, we should stage on RCs, and release once vote passed.